test/regression/44.test.kt

```diff
@@ -0,0 +1,15 @@
+// SYNTAX TEST "source.kotlin" "Keywords"
+
+  val test = !isC
+//^^^ source.kotlin storage.type.kotlin
+//   ^^^^^^ source.kotlin
+//         ^ source.kotlin keyword.operator.assignment.kotlin
+//           ^ keyword.operator.logical.kotlin
+//            ^^^ source.kotlin - keyword.operator.kotlin
+
+  val test2 =!isC
+//          ^ keyword.operator.assignment.kotlin
+//           ^^^ - keyword.operator.kotlin
```

Comment:
@nishtahir I don't get it, this test is testing for the bad case, not what the expectation is. `isC` is a variable name.

Reply:
This is testing that the `keyword.operator.kotlin` scope is not applied that's what the `-` sign indicates

Reply:
Aaaaah 😅